dCommon/eBlueprintSaveResponseType.h

```diff
@@ -0,0 +1,26 @@
+#pragma once
```

Comment:
should this go ahead and be moved to the enum dir even though that PR isn't mergd?

Reply:
Personally I would say no since it would be more efficient to not have merge conflicts but I am leaning more towards taking care of it in that PR